awswrangler/athena/_read.py

```diff
@@ -1042,26 +1024,12 @@ def read_sql_query(  # pylint: disable=too-many-arguments,too-many-locals
         If integer is provided, specified number is used.
     boto3_session : boto3.Session(), optional
         Boto3 Session. The default boto3 session will be used if boto3_session receive None.
-    max_cache_seconds : int
-        awswrangler can look up in Athena's history if this query has been run before.
-        If so, and its completion time is less than `max_cache_seconds` before now, awswrangler
-        skips query execution and just returns the same results as last time.
+    athena_cache_settings: typing.AthenaCacheSettings, optional
```

Comment:
I think we could add an example of how to set the cache settings either here or via a note/example in the docs?